web/templates/introduce_zulip_view_modal.hbs

```diff
@@ -0,0 +1,38 @@
+<div id="introduce-zulip-view-modal">
+    <p>
+        {{#if (eq zulip_view "inbox")}}
+            {{#tr}}The <b>inbox</b> view provides an overview of your conversations with unread messages.{{/tr}}
+        {{/if}}
+        {{#tr}}
+            In Zulip, a conversation is either a <z-link-direct-message>direct message</z-link-direct-message>
+            thread, or a <z-link-stream-topic>topic in a stream</z-link-stream-topic>.
+            {{#*inline "z-link-direct-message"}}<a href="/help/direct-messages" target="_blank" rel="noopener noreferrer">{{> @partial-block }}</a>{{/inline}}
+            {{#*inline "z-link-stream-topic"}}<a href="/help/streams-and-topics" target="_blank" rel="noopener noreferrer">{{> @partial-block }}</a>{{/inline}}
+        {{/tr}}
+        {{#if (eq zulip_view "inbox")}}
+            {{t 'The colored bars show where each conversation took place.' }}
+        {{/if}}
+    </p>
+    <p>
+        {{t "Click on a conversation you're interested in to view it."}}
+        {{#tr}}
+            To return here, you can:<z-return-options></z-return-options>
+            {{#*inline "z-return-options"}}
+                <ul>
+                    <li>{{#tr}}Use the <b>back</b> button in your browser or desktop app.{{/tr}}</li>
+                    <li>
+                        {{#if (eq zulip_view "inbox")}}
+                            {{#tr}}
+                                Click <z-icon-inbox></z-icon-inbox> <b>Inbox</b> in the left sidebar.
+                                {{#*inline "z-icon-inbox"}}<i class="zulip-icon zulip-icon-inbox" aria-hidden="true"></i>{{/inline}}
+                            {{/tr}}
+                        {{/if}}
+                    </li>
+                    {{#if current_home_view_and_escape_navigation_enabled}}
+                        <li>{{#tr}}Use <kbd>Esc</kbd> to go to your home view.{{/tr}}</li>
+                    {{/if}}
+                </ul>
+            {{/inline}}
```

Comment:
Why is this an `inline` block? I would have expected to just have "To return here, you can:" be on `tr` block, and then the rest of this just inside that.

Reply:
Yes, that'd be more simplistic.